utils/webpackCompiler.js

```diff
@@ -19,6 +19,11 @@ module.exports = (webpackConfig, cb) => {
     if (stats.hasErrors()) {
       logger.error(`${type} build failed\n`, stats.toString());
       logger.info('See webpack error above');
+
+      // Quit process if we're building
+      if (action === "build") {
+        process.exit(1);
+      }
```

Comment:
The callback `cb` is already being called with the `stats` object, so instead of having to pass in an extra parameter,`action` here, I think it might be cleaner to just have the upstream code and the process.exit (e.g. check for `stats.hasErrors()` in the `dev` action). This way the process management stays in the actions and the underlying utility functions stay pure. What do you think?